src/VacuumFields.jl

```diff
@@ -41,8 +41,8 @@ include("elliptic.jl")
 include("green.jl")
 
 include("control.jl")
-export AbstractControlPoint, FluxControlPoint, SaddleControlPoint, IsoControlPoint
-export FluxControlPoints, SaddleControlPoints, IsoControlPoints
+export AbstractControlPoint, FluxControlPoint, SaddleControlPoint, IsoControlPoint, FieldControlPoint
+export FluxControlPoints, SaddleControlPoints, IsoControlPoints, IsoRefControlPoints, FieldControlPoints
```

Comment:
Need to remove the vestigial IsoRefControlPoints export. Otherwise this is ready for merge.

Reply:
Fixed with [6056256](https://github.com/ProjectTorreyPines/VacuumFields.jl/pull/68/commits/60562569250f28e59426aa9da138053732247b07)